openedx/core/djangoapps/catalog/utils.py

```diff
@@ -109,7 +126,7 @@ def get_program_types(name=None):
         return []
 
 
-def get_programs_with_type(types=None, include_hidden=True):
+def get_programs_with_type(include_hidden=True):
```

Comment:
You'll need to update the docstring of this method since `types` are no longer a parameter.